### ISSUE TITLE
Can hooks have "name"s ?

### ISSUE THREAD

**Issue by schuhschuh** [openmole/openmole#21]
When I try to set the name of a ```AppendToCSVFileHook```, I get the error
```
<console>:160: error: inferred type arguments [_60.type] do not conform to method :='s type parameter bounds [V <: AnyRef{def setName(name: String): Any}]
             name      := "writeDiceToCsvFileHook",
```

**Comment by romainreuillon** [comment]
For now only tasks have names...

**Comment by romainreuillon** [comment]
Not sure what to do about that yet... may be we should be able to name everything...

**Comment by schuhschuh** [comment]
Ok, I just thought because there is a ```setName``` method mentioned in the error that it may be possible already to also name hooks. The reason I wanted to add a name is because I had some formal validation errors where it referred to the hook as "misc <type>". If this message would instead contain the name of a hook, it would be more obvious which particular hook failed the validation.

**Comment by romainreuillon** [comment]
Solved in a6363424e06e15c3dbf8780edeee4911a7670a45
